cost-model/src/cost_tracker.rs

```diff
@@ -888,7 +893,7 @@ mod tests {
         // assert cost are adjusted down
         let reduced_units = 3;
         expected_cost -= reduced_units;
-        cost_tracker.update_execution_cost(&tx_cost, cost - reduced_units);
```

Comment:
Should we add a similar `test_update_data_cost()` test?

Reply:
Yea, c4e26c684e46939888d26433f1faabed7f94e9df refactored this test covers combinations of adjustments.